webcompat/helpers.py

```diff
@@ -489,3 +491,36 @@ def add_csp(response):
         "style-src 'self' 'unsafe-inline'; " +
         "report-uri /csp-report"
     )
+
+
+def cache_policy(private=True, uri_max_age=86400, must_revalidate=False):
```

Comment:
@karlcow you might want to take a look at https://github.com/opendatateam/udata/pull/802/files#diff-74fa3771bbb3507e52e7f20e6ed64351R22 ;)